exporters/mongodb/mongodb.json.tmpl

```diff
@@ -0,0 +1,53 @@
+{
+    "exec": [
+        "{{ .exporter_binary_path }}",
+
+        {{ if .config.mongodb_uri}}
+        "--mongodb.uri",
+        "{{.config.mongodb_uri}}",
+        {{end}}
+
+        {{ if .config.collection_filters}}
+        "--mongodb.collstats-colls",
+        {{.config.collection_filters}},
+        {{end}}
+
+        {{ if .config.index_filters}}
+        "--mongodb.indexstats-colls",
+        {{.config.index_filters}},
+        {{end}}
+
+        {{if or (.config.database_stats) (eq .config.database_stats nil)}}
```

Comment:
In which cases will this condition be `false`?

Reply:
Got it, for a moment I thought `.config.database_stats` was a dict and got me confused. Everything makes sense now!